client/lib/signup/step-actions.js

```diff
@@ -424,7 +425,7 @@ export function createSite( callback, { themeSlugWithRepo }, { site }, reduxStor
 	const data = {
 		blog_name: site,
 		blog_title: '',
-		public: -1,
+		public: abtest( 'privateByDefault' ) === 'private' ? -1 : 1,
```

Comment:
Do we want _all_ new sites to enter the ab test or only the ones coming through the `main` flow?

Reply:
I don't see why not all...